src/Appwrite/Utopia/Response/Model/Rule.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+use stdClass;
+use Utopia\Database\Document;
+
+class Rule extends Model
```

Comment:
not sure what's confusing in response.php. I think it's fine as just Rule, just like it's Collection and not DatabaseCollection.

Reply:
For now this is fine, in the future we can add prefix. I would also recommend to use the directory as the prefix in the future instead of making the name longer.